ackhandler/sent_packet_handler.go

```diff
@@ -16,8 +16,12 @@ const (
 	// Maximum reordering in time space before time based loss detection considers a packet lost.
 	// In fraction of an RTT.
 	timeReorderingFraction = 1.0 / 8
+	// The default RTT used before an RTT sample is taken.
+	defaultInitialRTT = 100 * time.Millisecond
```

Comment:
This is duplicate with the const in the rttstats. Can we change SmoothedRTT to return that there if the value is zero?

Reply:
`SmoothedRTT` is used a lot all across quic-go, including the congestion, and compared for 0 at a few places. Changing the return value will cause all those checks to fail, so I don't think this is something we can do.

Reply:
It's not actually used a lot. I only see the use in the flowcontroller (where a default initial value would make sense), BandwidthEstimate() (which isn't called from anywhere and could be removed), and RetransmissionDelay() (where a too-low initial default doesn't matter). I'll leave it up to you, but I find it very strange to have the same const in two places.